composites/Plugin/Shared/components/Collapsible.js

```diff
@@ -65,6 +66,7 @@ const StyledTitle = styled.span`
 	white-space: nowrap;
 	text-overflow: ellipsis;
 	overflow-x: hidden;
+	color: ${ colors.$palette_grey_dark };
```

Comment:
Please never use a `$palette_...` color. Use `$color_...` instead. In this case: `$color_headings`